app/scripts/lib/user-agent.js

```diff
@@ -31,6 +31,15 @@ define(function (require, exports, module) {
       },
 
       /**
+       * Check if the browser is Mobile Safari.
```

Comment:
Can you add [a test for this](https://github.com/mozilla/fxa-content-server/blob/master/app/scripts/lib/user-agent.js)?